### DISCUSSION TITLE
Unable to escalate an alert from Security Onion Console

### DISCUSSION THREAD

**Discussion by Shorider** [Security-Onion-Solutions/securityonion#6740] [General]
I am currently running Security Onion 2.3.90

I have a setup of a master, a forward search node and logger.  I am also receiving traffic from a PFsense box. 

I can see logs coming in from different sensors via the security onion master console.  I log into the Security Onion Console and bring up the Alerts.   I get an issue where I am unable to escalate an alert when I click on the blue triangle with the ! symbol.  As soon as I click on it, I get a red banner at the top of the console stating:

Request failed with status code 400
![Escalate Error](https://user-images.githubusercontent.com/92332925/147966993-7e9bf8a0-5bf0-49ff-8aa1-6d114ddcb995.png)

I have tried multiple alerts and receive the same message.  I am still trying to learn the basic-intermediate level flows of security onion, but I am unable to figure this issue out.  I am showing that everything is running fine with so-allow.  I am able to open the direct link to TheHive as well with no issues.

Any thoughts?

**Comment by defensivedepth** [comment]
@Shorider Edited your question to remove the screencapture, as it contained personal emails / details. Please upload it again with that information removed.

**Comment by dougburks** [comment]
Have you checked the logs in /opt/so/log/ for additional clues?
